testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -509,5 +510,7 @@
 ITP_no_endif = resource_filename(__name__, 'data/no_endif_spc.itp')
 
 NAMDBIN = resource_filename(__name__, 'data/adk_open.coor')
+
+CRYST1_BOX = resource_filename(__name__, 'data/6cvm.pdb')
```

Comment:
So it might be best to add this with the rest of the `PDB` named filed. Also it might be pedantic but I would instead consider using something like `PDB_CRYOEM_BOX` instead of `CRYST1_BOX`. The former will help future developers understand why the file was included (and how it can be re-used).